### DISCUSSION TITLE
Idea: Notification icon

### DISCUSSION THREAD

**Discussion by stpierrevincents** [tuskyapp/Tusky#3150] [Ideas]
Hello,

I have fallen in love with Tusky. It is my favourite app for going onto my instance, mastodon.world, and I find it quite excellent to experience the fediverse with.

I have one issue, however: the notification icon... looks like a surprised cat, with a very long neck. Allow me to share a screenshot from my Pixel 6 Android device. It is below.

![unnamed (10)](https://user-images.githubusercontent.com/104039202/211176519-51c9723a-e724-4bd2-bc1c-9839a52800df.png)

It might just be me. Oftentimes, it is.

But, as you can see above, I believe the original intention is for it to be a trunk, with three dashes at its peak to highlight that it's making a loud noise from said trunk. The issue from this... is that, with it being such an intricate notification, it's easy to misread the notification. I think there is an opportunity to create a simpler notification.

I would offer an alternative if it pleases the contributors and others. I'm new to GitHub, so hopefully, I am placing this suggestion in the right place (I... already [made an issue of myself in the issues side](https://github.com/tuskyapp/Tusky/issues/3062)). I reviewed [the material design documentation](https://m2.material.io/design/iconography/system-icons.html#grid-and-keyline-shapes), and I believe I might have the beginnings of a reasonable alternative.

It's (1) simple, (2) an exclaimation mark which is contorted into a tusk, and (3) can reasonably be viewed at multiple sizes.

![Tusky-Notification on white](https://user-images.githubusercontent.com/104039202/211181335-c8d4fdf3-976f-4673-b2ab-98fbbb355294.png)

Below is the first rendering.

![unnamed (11)](https://user-images.githubusercontent.com/104039202/211181308-d45e2da0-3809-4208-b1c5-ca39740eef6e.png)

All the best, and looking to hear from you soon,
Vincent

**Comment by scottiefalkon** [comment]
The current notification icon is merely a small version of the app icon.
